packages/seacas/libraries/ioss/src/exodus/Ioex_DecompositionData.C

```diff
@@ -291,10 +291,6 @@ namespace Ioex {
             "exodus", filename, Ioss::READ_RESTART, Ioss::ParallelUtils::comm_self(), properties);
         Ioss::Region region(dbi, "line_decomp_region");
 
-        int status = Ioss::DecompUtils::line_decompose(
```

Comment:
The correct fix is to either check the status and report an erorr/warning if needed, or to remove just the `int status` part.  The function call is needed.

Reply:
Yes, I closed my PR b/c there were issues with it.  It would be best if you could create a PR with the optimal fix.